example-playbooks/replace_node/vars/main.yml

```diff
@@ -30,6 +30,10 @@ new_node_repair_timeout_seconds: 36000
 # Default bootstrap timeout for the new node
 new_node_bootstrap_wait_time_sec: 25200
 
+# If you set this to true, this playbook will allow you to replace an alive node
+# by draining/stopping the node being replaced before starting the replace
+alive_node_replace: false
```

Comment:
I don't see a logic we discussed before: if `alive_node_replace == false` you wanted to check if we are not replacing an alive node my mistake: check if scylla service is running and if "yes" - fail angrily!

Reply:
We were already doing that, so all I had to add was the task responsible for killing scylla if this flag is set to true.